test/unit/puppetlabs/services/master/master_core_test.clj

```diff
@@ -232,15 +232,18 @@
                      "modules/foo/files/bar"
                      "modules/foo/files/bar/baz.txt"
                      "modules/foo/files/bar/more/path/elements/baz.txt"
-                     "modules/foo/files/bar/%2E%2E/baz.txt"]
+                     "modules/foo/files/bar/%2E%2E/baz.txt"
+                     "environments/production/files/~/.bash_profile"
```

Comment:
No totally. Absolutely valid. Also totally a path that looks like someone messed up a `cp -r` somewhere.